cmd/bolt/main.go

```diff
@@ -593,12 +645,21 @@ func (cmd *PageCommand) PrintFreelist(w io.Writer, buf []byte) error {
 	p := (*page)(unsafe.Pointer(&buf[0]))
 
 	// Print number of items.
-	fmt.Fprintf(w, "Item Count: %d\n", p.count)
+	fmt.Fprintf(w, "Item Count Field: %d\n", p.count)
+	fmt.Fprintf(w, "Overflow: %d\n", p.overflow)
+
+	idx, count := 0, int(p.count)
+	if p.count == 0xFFFF {
+		idx = 1
+		count = int(((*[maxAllocSize]pgid)(unsafe.Pointer(&p.ptr)))[0])
+		fmt.Fprintf(w, "Item Count of 65535 (max uint16) indicates a freelist overflow. Actual Item Count: %d\n", count)
```

Comment:
sort of wordy/obscure, maybe only have `Item Count:` but determine the count depending on whether it's overflow or not? if the physical dump of the page structure is needed it should probably be handled separate from this